### PR TITLE
Add new case for virsh domdisplay's new option '--type'.

### PR DESCRIPTION
**Pull request by liuzzfnst**

### CODE REVIEW

**Comment by waynesun09** [comment]
commit 6215d54 also mixed in here, and you need check Travis CI failure which shows pep8 failure.

**Comment by liuzzfnst** [comment]
Yes， I will fix this.

File: libvirt/tests/src/virsh_cmd/domain/virsh_domdisplay.py [hunk 1]
```diff
@@ -34,6 +35,14 @@ def run(test, params, env):
     vmxml_backup = vm_xml.VMXML.new_from_inactive_dumpxml(vm_name)
     tmp_file = os.path.join(test.tmpdir, "qemu.conf.bk")
 
+    if "--type" in options:
+        if not libvirt_version.version_compare(1, 2, 6):
+            raise error.TestNAError("--type option is not supportted in this"
```
Comment (PandaWei) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domdisplay.py]: How do you confirm the version?
It's better to add the URL of commits related to "--type",
which is useful for review.

**Comment by PandaWei** [comment]
ACK

**Comment by PandaWei** [comment]
Please attach the result of new case.
If all pass, Let's merge this PR

**Comment by liuzzfnst** [comment]
The test result on libvirt-1.2.13 fc20.x86_64

\ # ./run -k --no-downloads -t libvirt --tests virsh.domdisplay
SETUP: PASS (0.20 s)
DATA DIR: /var/lib/virt_test
DEBUG LOG: /root/autotest/client/tests/virt/logs/run-2015-09-10-13.29.02/debug.log
TESTS: 30
(1/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.option_include-passwd.negtive_readonly.spice.with_ssl: PASS (2.10 s)
(2/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.option_include-passwd.negtive_readonly.spice.without_ssl.domain_id: PASS (1.58 s)
(3/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.option_include-passwd.negtive_readonly.vnc.domain_uuid: PASS (1.60 s)
(4/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.option_include-passwd.readwrite.spice.with_ssl: PASS (2.25 s)
(5/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.option_include-passwd.readwrite.spice.without_ssl.domain_id: PASS (1.68 s)
(6/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.have_passwd.option_include-passwd.readwrite.vnc.domain_uuid: PASS (1.62 s)
(7/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.no_options.positive_readonly.spice.with_ssl: PASS (2.19 s)
(8/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.no_options.positive_readonly.spice.without_ssl.domain_id: PASS (1.65 s)
(9/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.no_options.positive_readonly.vnc.domain_uuid: PASS (1.66 s)
(10/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.no_options.readwrite.spice.with_ssl: PASS (2.17 s)
(11/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.no_options.readwrite.spice.without_ssl.domain_id: PASS (1.69 s)
(12/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.no_options.readwrite.vnc.domain_uuid: PASS (1.68 s)
(13/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_include-passwd.negtive_readonly.spice.with_ssl: PASS (2.10 s)
(14/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_include-passwd.negtive_readonly.spice.without_ssl.domain_id: PASS (1.65 s)
(15/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_include-passwd.negtive_readonly.vnc.domain_uuid: PASS (1.56 s)
(16/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_include-passwd.readwrite.spice.with_ssl: PASS (2.21 s)
(17/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_include-passwd.readwrite.spice.without_ssl.domain_id: PASS (1.69 s)
(18/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_include-passwd.readwrite.vnc.domain_uuid: PASS (1.69 s)
(19/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.vnc_type.spice.with_ssl: PASS (2.44 s)
(20/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.vnc_type.spice.without_ssl.domain_id: PASS (1.58 s)
(21/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.vnc_type.vnc.domain_uuid: PASS (1.75 s)
(22/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.spice_type.spice.with_ssl: PASS (2.19 s)
(23/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.spice_type.spice.without_ssl.domain_id: PASS (1.67 s)
(24/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.spice_type.vnc.domain_uuid: PASS (1.55 s)
(25/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.type_without_post.spice.with_ssl: PASS (2.08 s)
(26/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.type_without_post.spice.without_ssl.domain_id: PASS (1.55 s)
(27/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.type_without_post.vnc.domain_uuid: PASS (1.56 s)
(28/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.none_type.spice.with_ssl: PASS (2.19 s)
(29/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.none_type.spice.without_ssl.domain_id: PASS (1.67 s)
(30/30) type_specific.io-github-autotest-libvirt.virsh.domdisplay.no_passwd.option_type.none_type.vnc.domain_uuid: PASS (1.68 s)
TOTAL TIME: 54.75 s
TESTS PASSED: 30
TESTS FAILED: 0
SUCCESS RATE: 100.00 %

**Comment by PandaWei** [comment]
Thanks @liuzzfnst  @waynesun09 
Merge now

**Pull request merged**